src/Tribe/Aggregator/Record/CSV.php

```diff
@@ -341,6 +341,14 @@ public function reset_tracking_options() {
 	}
 
 	public function continue_import() {
+		global $wpdb;
+		if ( empty( $wpdb->get_var( "SELECT GET_LOCK('tribe_ea_csv_import',10)" ) ) ) {
+			do_action( 'tribe_log', 'debug', 'CSV Import',
+				[ 'message' => 'Could not acquire lock.', 'error' => $wpdb->last_error ] );
```

Comment:
* :no_entry_sign: Multi-line function call not indented correctly; expected 12 spaces but found 16
* :no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 7813850d48168b0e2e87143de8008e99</sub>